cmd/maya-exporter/app/collector/cstorcollector.go

```diff
@@ -167,7 +169,19 @@ func (c *Cstor) set(m *Metrics) error {
 		newResp.Iqn,
 		"localhost",
 		"cstor",
+		volStats.status,
 	).Set(volStats.uptime)
+
+	for i := 0; i < int(volStats.replicaCount); i++ {
+		fmt.Fprintf(&replicaAddress, "%s,", volStats.replicas[i].Address)
```

Comment:
Why dont we use StringBuilder methods?
Will there be a extra comma?

Reply:
Will look into it, yes there will be an extra comma.

Reply:
updated in the recent commit.